mne/inverse_sparse/mxne_inverse.py

```diff
@@ -638,7 +638,7 @@ def tf_mixed_norm(evoked, forward, noise_cov, alpha_space, alpha_time,
     X, active_set, E = tf_mixed_norm_solver(
         M, gain, alpha_space, alpha_time, wsize=wsize, tstep=tstep,
         maxit=maxit, tol=tol, verbose=verbose, n_orient=n_dip_per_pos,
-        log_objective=True, debias=debias)
+        debias=debias)
```

Comment:
I suggest to have `log_objective=log_objective` here and add `log_objective=10` as a new parameter to `tf_mixed_norm`. The user might want to control the frequency of dgap evaluations.

Reply:
We can't have  `log_objective=log_objective` here since `log_objective` is not an argument of `tf_mixed_norm()`. I suggest we have `log_objective=None`, and `dgap_freq=dgap_freq` with `dgap_freq` added to `tf_mixed_norm()`.